model/src/main/proto/deprecation.proto

```diff
@@ -0,0 +1,42 @@
+syntax = "proto3";
+
+package model;
+
+import "version.proto";
+
+option java_package = "org.oppia.android.app.model";
+option java_multiple_files = true;
+
+// Top-level proto used to store deprecation messages
+message DeprecationResponseDatabase {
+    // Stores the response to a request for an optional update.
+    DeprecationResponse app_deprecation_response = 1;
+
+    // Stores the response to a dialog notifying the user of an OS deprecation.
+    DeprecationResponse os_deprecation_response = 2;
```

Comment:
Do we only ever expect to store one of each response at a time? This database would only contain 2 items at a time, is this by design?

I have added a comment in `DeprecationNoticeType` below that provides an alternative.

Reply:
I had originally designed a database similar to the one on profile, one which can take a map of responses. However, we had a discussion with Ben where he suggested the new approach of using only two responses because that's all we are going to need. No need keeping responses from 10 versions ago, which might not be used. We only keep a response for the latest app version.

Reply:
Okay, makes sense.